src/org/zaproxy/zap/extension/pscanrulesAlpha/CacheableScanner.java

```diff
@@ -498,7 +499,7 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 				lifetimeFound = true;
 				lifetime = 60 * 60 * 24 * 365;
 				//a liberal heuristic was assumed, for which no actual evidence exists
-				freshEvidence = Constant.messages.getString(MESSAGE_PREFIX_STORABLE_CACHEABLE + "evidence.liberallifetimeheuristic");
```

Comment:
The variable freshEvidence should be set to null (or an empty String), otherwise the alert might end up with wrong evidence.